app/assets/stylesheets/common/base/login-signup-page.scss

```diff
@@ -13,10 +13,24 @@ body.login-page,
 body.signup-page,
 body.invite-page,
 body.password-reset-page {
-  & ~ .powered-by-discourse,
+  .powered-by-discourse,
   .above-main-container-outlet {
     display: none;
   }
+  #main-outlet {
+    padding: 0;
+  }
```

Comment:
This overrides the default `padding-top: 1.5em` rule only, right? Maybe we don't need it or we limit the override to `padding-top`? 

Just being wary of adding too many overrides for these pages, it makes theming trickier because themers don't necessarily check that these pages will work fine. 

Also, in some places we do: 

```
html.footer-nav-visible {
  #main-outlet {
    padding-bottom: calc(var(--footer-nav-height) + 15px);
  }
}
```

That applies to DiscourseHub on mobile + iPad (but if we end up with an app footer, it will apply everywhere). Not sure if the override here resets that... 

Ah, also this: 

```
#main-outlet {
  @media only screen and (orientation: landscape) {
    padding-right: env(safe-area-inset-right);
    padding-left: env(safe-area-inset-left);
  }
}
```

Reply:
Good points to think on. I believe the consensus was to remove footer & other content from rendering on signup/login pages. Padding being set to 0 here also allows for more visibility of the content, as these pages are not displaying content as the main app does.